app/views/users/edit.html.erb

```diff
@@ -1,6 +1,6 @@
-# <h1>Editing User</h1>
+<h1>Editing User</h1>
```

Comment:
It looks like we removed the #edit method in the controller so is this file still needed?

Reply:
This file was not needed and was removed